qtcreator/cmut__qtcreator__import_qml.cmake

```diff
@@ -0,0 +1,58 @@
+# In order to use qtcreator inspector to load modules QML dynamically
+# and propose code completion in multi libraries environment we have to export
+# QML files and set QML_IMPORT_PATH variable.
+
+function(cmut__qtcreator__import_qml target)
+
+    get_target_property(_target_dependencies ${target} LINK_LIBRARIES)
+
+    __cmut__qtcreator__get_qml_import_paths(_qml_directories "${_target_dependencies}")
+
+    set( QML_IMPORT_PATH "${_qml_directories}" CACHE STRING "Qml import path" FORCE)
```

Comment:
set QML_IMPORT_PATH as INTERNAL data to not see it in cmake gui, thought?

Reply:
Done